### PR TITLE
feature: Implemented Sentry hooks and logging - closes #94 

### LINKED ISSUE #94: feature: implement sentry hooks
# User Story
Cloudinventario has no live error reporting right now, so its not easy to watch for errors.

# Tasks
* Implement sentry client (enabled only if config is present)
* Take sentry configuration from environment (SENTRY_*) and config file (in this order)

### PR DESCRIPTION
**Pull request by MatejHor**

### CODE REVIEW

File: cloudinventario [hunk 1]
```diff
@@ -86,6 +91,27 @@ def collect(data):
      setproctitle.setproctitle(proctitle)
    return False
 
+# sentry load and apply config
+def sentryConfig(config, level):
+  dsn = (os.getenv("SENTRY_DSN") or config['sentry']['dns'])
```
Comment (samsk) [inline cloudinventario:96]: Typo: its `dsn` not `dns`

File: cloudinventario [hunk 2]
```diff
@@ -86,6 +91,27 @@ def collect(data):
...
+  environment= (os.getenv("SENTRY_ENVIRONMENT") or config['sentry']['env'])
+  traces_sample_rate= float(os.getenv("SENTRY_TSR") or config['sentry']['tsr'])
+  event_level= int(os.getenv("SENTRY_EVENT_LEVEL") or config['sentry']['event_level'])
```
Comment (samsk) [inline cloudinventario:99]: Init will fail if these optional config keys are not present, plz provide sensible default to make them really optional. Only `dsn` is required, if that is not present - do not init.

**Review by samsk — CHANGES_REQUESTED** [review]

File: cloudinventario (hunk 1 above)
Comment (MatejHor) [reply cloudinventario]: Sorry for the mistake in spelling. In '4a7f19b' commit spelling is fixed.

File: cloudinventario (hunk 2 above)
Comment (MatejHor) [reply cloudinventario]: In '4a7f19b' commit parameters are really optional.  Config if not found ENV return 'dev', TSR return '1.0' and EVENT_LEVEL return logging.ERROR.

**Pull request merged**